tools/wdio/plugins/custom-commands.mjs

```diff
@@ -120,14 +120,12 @@ export default class CustomCommands {
      * to a specific value. Default is to set the header to one hour in the past.
      */
     browser.addCommand('mockDateResponse', async function (serverTime = Date.now() - (60 * 60 * 1000), opts = {}) {
-      const { flags } = opts
+      const { flags = {} } = opts
+      const body = JSON.stringify(rumFlags(flags, { nrServerTime: serverTime }))
```

Comment:
what's with the serverTime formula here? should just use date.now default, no?

Reply:
The testing server always sets the timestamp 1 hour back by default, but we have some tests that change the value dynamically so it needs to be parameterized.

Reply:
none of the mockdateresponse actually sets any time

Reply:
ah thanks for catching that, then thats a bug with the server-time tests.  We will need to fix that in a separate PR

Reply:
https://new-relic.atlassian.net/browse/NR-314521?atlOrigin=eyJpIjoiNzliODMwOGFjMzVjNDU2Zjg2MDE1ZDc5NmIwZTZkMDgiLCJwIjoiaiJ9

Reply:
actually.. lets just fix in this PR.  I'll do that

Reply:
This would be a lot of effort to rewrite the tests to support before and after.  It is checked in a unit test, so i think this is fine to leave alone.  The parameter isnt hurting anything and can be used later if we need to dynamically set the timestamp.